docs/en/operations/table_engines/file.md

```diff
@@ -1,6 +1,7 @@
-# File(InputFormat) {#table_engines-file}
+# File {#table_engines-file}
 
-The data source is a file that stores data in one of the supported input formats (TabSeparated, Native, etc.).
+The File table engine keeps the data in a file in one of the supported [file
```

Comment:
@BayoNet checked on different PR that these extra line breaks mess up the resulting HTML, so let's keep paragraph as one long line for now.

Reply:
It doesn't seem to break plain text, but I found that it breaks Note:
https://clickhouse.yandex/docs/en/operations/settings/settings/#session_settings-input_format_defaults_for_omitted_fields

The second line is outside the Note block. I'll try to fix the generation of HTML.